### DISCUSSION TITLE
In got what is the equivalent option of { jar: true } in Request - Simplified HTTP client

### DISCUSSION THREAD

**Discussion by cheekykesav** [sindresorhus/got#1904] [Q&A] [answered]
In got what is the equivalent option of { jar: true } in **[Request - Simplified HTTP client](https://www.npmjs.com/package/request)** 

**Request** docs :
jar - if true, remember cookies for future use (or define your custom cookie jar; see examples section)
const request = request.defaults({jar: true})
request('http://www.google.com', function () {
  request('http://images.google.com')
})

Like the above option in **request**, is there any **got** equivalent to remembering cookies for all future API calls once login API is called at the beginning?

**Comment by ghost** [comment, marked as answer]
https://github.com/sindresorhus/got/blob/main/documentation/2-options.md#cookiejar

**Reply by susnick** [reply]
I thought got is supposed to have cookie support built in. Why does your link show an example requiring tough-cookie?
![image](https://user-images.githubusercontent.com/1975526/151056838-10929d92-de57-45c8-9352-9424c6bcb2be.png)

**Reply by ghost** [reply]
Because tough-cookie makes it easier. You can also write your own cookie jar, which isn't too hard and I've done it before

**Reply by szmarczak** [reply]
> Why does your link show an example requiring tough-cookie?

How do you want to store cookies otherwise?
